client/spec/fetch_spec.js

```diff
@@ -77,19 +81,26 @@ describe('Fetch', function() {
         workspace.switchToDesk('SPORTS DESK').then(content.setListView);
         expect(element.all(by.repeater('items._items')).count()).toBe(2);
 
-        browser.get('/#/workspace/ingest');
-        workspace.switchToDesk('SPORTS DESK');
-        content.setListView();
-        content.actionOnItem('Fetch', 0);
-        browser.get('/#/workspace/content');
-        workspace.switchToDesk('SPORTS DESK');
-        content.setListView();
+        browser.get('/#/workspace/ingest').then(function() {
```

Comment:
it was but the thing what no one of developers which were trying this issue before managed to find the reason of that "desynchronization", so if i won't manage to find the core problem of that than maybe we can consider moving to angular-independed e2e framework, like http://nightwatchjs.org/ or http://dalekjs.com/pages/getStarted.html